build.gradle

```diff
@@ -52,7 +52,7 @@ sourceSets {
         resources {
             srcDir 'src/blockout/resources'
         }
-        compileClasspath += sourceSets.api.compileClasspath
+        // compileClasspath += sourceSets.api.compileClasspath
```

Comment:
This is needed. It does not at a depencency to the api module. But to the dependencies of the API module. AKA Minecraft and such.